.codespellrc

```diff
@@ -0,0 +1,8 @@
+[codespell]
+skip = *.po,_build,_static,porting_guide*,collections,style_guide,
+count =
+check-filenames = true
+quiet-level = 3
+ignore-words = .codespellignorewords
+exclude-file = .codespellignorelines
+builtin = clear,rare,names,en-GB_to_en-US,informal,code
```

Comment:
```suggestion
builtin = 
    clear
    code
    en-GB_to_en-US,
    informal
    names
    rare
```

(I sorted the items in the process)

Reply:
@gotmax23 Unfortunately it looks like codespell only respects the comma separated list (with no space characters).

Reply:
Yeah, the config parsing code is very naive. Let's leave it as is.

/resolve